src/components/header/NavBar.vue

```diff
@@ -57,16 +72,19 @@ const hasNext = computed<boolean>(() => {
   }
   return true;
 });
-const labels = computed<Labels>(() => configStore.config.labels || {
-  manifest: 'manifest',
-  item: 'item',
-});
-const nextButtonLabel = computed<string>(() => (itemIndex.value === manifest.value.sequence.length - 1
-  ? `${t('next')} ${t(labels.value.manifest ? labels.value.manifest : 'Manuscript')}`
-  : `${t('next')} ${t(labels.value.item)}`));
+
+// variables are named according to the general concept of ie (page = sheet, item);  (document = manuscript)
```

Comment:
page and document are not according to general concept. They are only used in GFL. Instead we use item and manifest.